crates/schemalate/src/firebolt/firebolt_projections.rs

```diff
@@ -2,11 +2,17 @@ use std::collections::BTreeMap;
 use std::iter::FromIterator;
 
 use json::schema::types;
+use lazy_static::lazy_static;
 use protocol::flow::{inference::Exists, materialization_spec, FieldSelection};
 use protocol::materialize::{constraint, validate_request, Constraint};
+use regex::Regex;
 
 use crate::firebolt::errors::BindingConstraintError;
 
+lazy_static! {
+    static ref VALID_FIELD_REGEX: Regex = Regex::new(r"^[a-zA-Z_][a-zA-Z0-9_]*$").unwrap();
```

Comment:
Are unicode letters / numbers allowed by firebolt? Or strictly ASCII ?

If they are, `\p{Letter}` and `\p{Digit}` are the equivalent character groups for a-z and 0-9.

Reply:
Apparently ASCII only: 
<img width="652" alt="image" src="https://user-images.githubusercontent.com/2807772/160624176-17ed3517-7733-4e8d-bced-07444ba0d360.png">